studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Sharing your graph
+---
+
+import {
+  ExpansionPanel,
+} from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
+
+Apollo Studio provides features that help you share your graph with both teammates and external consumers.
+
+
+## Making a graph public
+
+You can make individual [variants](./org/graphs/#managing-variants) of your graph public. If you do, people _outside_ your organization can view the following pages for that variant in Studio:
```

Comment:
@StephenBarlow Do you think it is sufficiently clear that any unauthenticated user on the internet who has access to the public graph link will be able to open the graph schema? Not sure if it is just because I have been living in it for so long that this might make me wonder if I have more than 1 Studio org, then it is *not* just that an authenticated user of another org can access, it is any unauthenticated user.

Reply:
To clarify - any unauthenticated user with the public graph link will be able to access the schema